packages/api/models/granules.js

```diff
@@ -251,7 +251,7 @@ class Granule extends Manager {
 
     await this.updateStatus({ granuleId: granule.granuleId }, 'running');
 
-    await Lambda.invoke(process.env.invoke, lambdaPayload);
+    await Lambda.invoke(process.env.invoke, lambdaPayload).catch((error) => logger.error(error));
```

Comment:
Ended up adding an on error event listener to Lambda.invoke instead of trying to deal with the error here. Let me know if you think that should be changed

Reply:
good idea.